MarketplaceService/Startup.cs

```diff
@@ -39,7 +42,10 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<IDAppService, DAppService>();
             //Controllers
             services.AddControllers();
-            
+            //Message Consumer
+            services.AddMessageConsumer(Configuration["MessageQueueSettings:Uri"],
+                "marketplace-service",
```

Comment:
Hier moet de messagequeue naam aangepast worden zodat hij overeenkomt me hoe de EmailService geschreven is